src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+{
+
+	"death.attack.mmorpg": "%1$s был убит %2$s",
+	"block.mmorpg.harvest.purple": "Хаотические семена",
+	"block.mmorpg.harvest.green": "Первозданные семена",
+	"block.mmorpg.harvest.blue": "Прозрачные семена",
+	"item.mmorpg.custom_loot_crate": "Ящик с добычей",
+	"item.mmorpg.loot_crate": "Ящик с добычей",
+	"item.mmorpg.custom_crate": "Ящик с экипировкой",
+
+"_comment": "# CREATIVE TAB",
+
+	"itemGroup.mmorpg.tab": "Mine and Slash",
+
+	"death.attack.mmorpg.custom_damage": "Много урона!",
+	"death.attack.death_potion": "%1$s подумал, что это отличная идея выпить зелье с названием 'Смерть'.",
+
+"_comment": "STATIONS",
+
+	"block.mmorpg.teleporter": "Картографическое устройство",
+
+"_comment": "ITEM TOOLTIPS",
+
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
+	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
+	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
+	"mmorpg.tooltip.capacitor2": "Множитель топлива",
+	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо слота",
+
+"_comment": "RARITY ITEM NAMES",
+
+"_comment": "MISC ITEMS",
+
+	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
+	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
+	"item.mmorpg.jewel.int": "Звездная пыль",
```

Надо придерживаться единобразия, чтобы везде было "<прилагательное> самоцвет"
```suggestion
	"item.mmorpg.jewel.int": "Звездный самоцвет",
```

Из других вариантов: Космический, Астральный, Небесный

---

Исправлено